### PR TITLE
Fix docker compose commands in readme and improve wording

### PR DESCRIPTION
**Pull request by Famezzs**

### CODE REVIEW

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

File: README.md [hunk 1]
```diff
@@ -64,21 +65,26 @@ Technologies used:
...
+### Running the project out of the box   
+
+This application has everything needed for hosting it, though commands below require Docker.  
+If you want to run latest version of this API:  
+- `docker compose -f docker-compose.api-pull.yml up -d` to start  
+- `docker compose -f docker-compose.api-pull.yml down` to stop  
+
+If you are making changes to the code:  
+- `docker-compose -f docker-compose.api-build.yml up -d` to start  
+- `docker-compose -f docker-compose.api-build.yml up -d` to stop
```
Comment (ram333n) [inline README.md:77]: typo?

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

**Comment by knurff** [comment]
> There is no coverage information present for the Files changed

|Total Project Coverage|51.05%|:green_apple:|
|:-|:-:|:-:|

**Review by ram333n — APPROVED** [review]

**Pull request merged**